src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -4602,7 +4811,5 @@ partial class C
             Assert.Equal("SourceFile(Program.cs[52..53))", defSymbol.Locations.Single().ToString());
             Assert.Equal("SourceFile(Program.cs[97..98))", implSymbol.Locations.Single().ToString());
         }
-
-        // PROTOTYPE(partial-properties): override partial property where base has modopt
```

Comment:
Unless there is some particular time pressure, I would favor spending a couple of days closing the remaining compiler questions/issues over punting them to a follow-up issue. They seem small and it will bring peace of mind (hopefully we don't have to revisit) and it leaves the code with fewer comments for follow-up work.

Remaining suggested tests from this morning:
- which signature do we emit in case of parameter name differences, xml docs referencing parameter names 
- SkipLocalsInit
- `new` modifier?
- split declaration (one has get, other has set) with one implementation

Reply:
If you feel like more time is needed to reach the right level of quality, that's OK. However, it is a feature I am very eager to get into our partners hands, so that they can adopt it and ship their updated generators with a decent amount of time for .NET 9. Also, I am going to be out next week, so won't be able to assist with merging the feature or getting it merge-ready during that time.

re: coverage.
- which parameter names we use in emit is tested in `IndexerParameterNameDifference`.
- xml docs parameter name difference is tested in `DocumentationCommentCompilerTests.PartialIndexer_Paramref_03` and `PartialIndexer_Paramref_04`.
- SkipLocalsInit test was added in `AttributeTests_WellKnownAttributes.SkipLocalsInit_PartialPropertyAccessor_ContainsLocalFunction`.
- `Semantics_ExtendedModifier` and `ExtendedDifference_01` tests `new`.
- Adding another split declaration test `DuplicateDeclaration_08_Definition`.
- Adding the modopt test `CustomModifierCopyTests.TestPartialPropertyOverrideCombinations`

Reply:
Thanks for the coverage update
I think it's better to wait until week of June 10th to button up and merge. But I'm okay if Jared is okay with merging with those known follow-ups. Then I would recommend we do the follow-up soon while things are "loaded" in our minds and preferably before 17.11 ships.

Reply:
Discussed offline, our plan is to merge to main after this PR is merged and to address the remaining items in #73772 in 17.11